py-polars/polars/selectors.py

```diff
@@ -31,6 +31,49 @@
         from typing_extensions import Self
 
 
+def expand_selectors(
```

Comment:
Should this be public? If so, does it need a doc entry?

Reply:
Hmm, probably not actually - good catch!
I'll prefix with an "_" and omit from "__all__" 👌